src/screens/Departures/components/EstimatedCallItem.tsx

```diff
@@ -179,7 +195,18 @@ function getA11yLineLabel(departure: any, t: any, language: any) {
 
   return `${
     departure.cancellation ? t(CancelledDepartureTexts.message) : ''
-  } ${a11yLine} ${a11yFrontText} ${a11yDateInfo}`;
+  } ${getLineA11yLabel(departure, t)} ${a11yDateInfo}`;
+}
+
+function getLineA11yLabel(departure: any, t: any) {
```

Comment:
I see `any` was used here from before, but why? 🤔 I think we can try to remove them now that we are touching the code.

Reply:
yes :)